simple_network_sim/network_of_populations.py

```diff
@@ -14,6 +14,17 @@
 
 # CurrentlyInUse
 def countInfectiousAgeStructured(dictOfStates, time):
+    """Count the number of infectious individuals in all states at some time.
```

Comment:
I think instead of *all states* it should be *all nodes and across all age groups* or something like that